ios_config.sh

```diff
@@ -62,7 +61,7 @@ function setPlistValue {
 
 function getJsonKeyValue () {
   if [[ ${_RN_ROOT_EXISTS} ]]; then
-    ruby -KU -e "require 'rubygems';require 'json'; output=JSON.parse('$1'); puts output[$_JSON_ROOT]['$2']"
+    ruby -KU -e "require 'rubygems';require 'json'; output=JSON.parse('$1'); puts output['expo']['extra']['react-native-google-mobile-ads']['$2']"
```

Comment:
The problem here is that some people are already removing the `expo` property in the `app.json` file, so, this code should actually first check if it `root > extra > react-native-google-mobile-ads` is present, if not, check `root > expo > react-native-google-mobile-ads`